projects/sandbox/deployment/test/ingress.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: sandbox-ui-ingress-test
```

Comment:
can you add the prefix `metis-` so `metis-sandbox-ui-ingress-test`.  
Same goes for all namings.  
That will be more descriptive for us and others.